spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -18,9 +31,16 @@
     context "on #{plat[:name]} we should behave ourselves" do
       let(:platform) { plat }
       let(:cur_plat) { Vanagon::Platform::DSL.new(plat[:name]) }
+      let (:wixtestfiles) { "/tmp/spec/fixtures/wix/resources/windows/wix" }
 
       before do
         cur_plat.instance_eval(plat[:block])
+        FileUtils.mkdir_p(wixtestfiles)
+        FileUtils.cp_r("spec/fixtures/wix/resources/windows/wix", "#{wixtestfiles}/..")
```

Comment:
What's the point of moving these files all over the place? Wouldn't it be easier to just copy them from `spec/fixtures/wix/resources/windows/wix` instead of moving them to `/tmp/spec/fixtures/wix/resources/windows/wix` and copying them around from there?